DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/SendBinding.cs

```diff
@@ -83,20 +92,40 @@ private async Task HandleSend(string modelCardId)
         throw new InvalidOperationException("No publish model card was found.");
       }
 
-      using IUnitOfWork<SendOperation> sendOperation = _unitOfWorkFactory.Resolve<SendOperation>();
+      using IUnitOfWork<SendOperation<ElementId>> sendOperation = _unitOfWorkFactory.Resolve<
+        SendOperation<ElementId>
+      >();
+
+      List<ElementId> revitObjects = modelCard.SendFilter.GetObjectIds().Select(id => ElementId.Parse(id)).ToList();
+
+      var sendInfo = new SendInfo(
+        modelCard.AccountId,
+        modelCard.ProjectId,
+        modelCard.ModelId,
+        _revitSettings.HostSlug,
+        _convertedObjectReferences,
+        modelCard.ChangedObjectIds
+      );
 
-      string versionId = await sendOperation.Service
+      var sendResult = await sendOperation.Service
         .Execute(
-          modelCard.SendFilter,
-          modelCard.AccountId,
-          modelCard.ProjectId,
-          modelCard.ModelId,
+          revitObjects,
+          sendInfo,
           (status, progress) => OnSendOperationProgress(modelCardId, status, progress),
           cts.Token
         )
         .ConfigureAwait(false);
 
-      Commands.SetModelCreatedVersionId(modelCardId, versionId);
+      // Store the converted references in memory for future send operations, overwriting the existing values for the given application id.
+      foreach (var kvp in sendResult.convertedReferences)
+      {
+        _convertedObjectReferences[kvp.Key + modelCard.ProjectId] = kvp.Value;
```

Comment:
we may want to see if we can get some logging showing the size of this dictionary since we are always adding items and never removing any.